data/net/minecraft/data/structures/SnbtToNbt.mapping

```diff
@@ -17,6 +19,20 @@ CLASS net/minecraft/data/structures/SnbtToNbt
 		COMMENT Performs this provider's action.
 		ARG 1 cache
 	METHOD storeStructureIfChanged (Lnet/minecraft/data/HashCache;Lnet/minecraft/data/structures/SnbtToNbt$TaskResult;Ljava/nio/file/Path;)V
-		ARG 1 directory
-		ARG 2 taskResult
-		ARG 3 path
+		ARG 1 cache
+		ARG 2 result
+		ARG 3 folderPath
+	CLASS Filter
+		METHOD apply (Ljava/lang/String;Lnet/minecraft/nbt/CompoundTag;)Lnet/minecraft/nbt/CompoundTag;
+			ARG 1 fileName
+			ARG 2 tag
+	CLASS TaskResult
+		METHOD <init> (Ljava/lang/String;[BLjava/lang/String;Ljava/lang/String;)V
+			ARG 1 name
+			ARG 2 payload
+			ARG 3 snbtPayload
+			ARG 4 hash
+	CLASS StructureConversionException
+		METHOD <init> (Ljava/nio/file/Path;Ljava/lang/Throwable;)V
+			ARG 1 path
```

Comment:
`pathMessage`

Reply:
Not sure about this one, it sounds misleading since the `path` goes through `toAbsolutePath` to generate the actual message, so if you provide a relative path then the exception message will be different.

Reply:
This can just be path then. It's an error so I'm okay with a generified version. Though this should probably get a javadoc on what the exception is for.